### PR TITLE
Automate release with bumpver.

### PR DESCRIPTION
**Pull request by yakutovicha**
Also, add a description to the readme and bumpver as an optional dependency.

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -22,12 +22,18 @@ jobs:
 
     - name: Upgrade setuptools and install package
       run: |
-        python -m pip install --upgrade pip setuptools
-        python -m pip install -e .
+        python -m pip install build
 
     - name: Build source distribution
       run: python ./setup.py sdist
```
Comment (eimrek) [inline .github/workflows/publish.yml]: Probably related to the last PR: does this work when we don't have setup.py any more?

**Review by eimrek — COMMENTED** [review]
Thanks, the bumpver looks convenient. The setup.py issue can be fixed (if needed) in a different PR, i guess

File: .github/workflows/publish.yml (hunk 1 above)
Comment (yakutovicha) [reply .github/workflows/publish.yml]: yes, `pyproject.toml` replaces both `setup.json/cfg` and `setup.py`. I wrote it in the commit description.

Comment (yakutovicha) [reply .github/workflows/publish.yml]: oh, wait, you are right - the file `setup.py` can't be called as it is absent 😄 . I will fix this.

Comment (yakutovicha) [reply .github/workflows/publish.yml]: Interesting thing to read: https://stackoverflow.com/questions/58753970/how-to-build-a-source-distribution-without-using-setup-py-file

**Review by eimrek — APPROVED** [review]
looks good, thanks!

**Pull request merged**